apps/gitness/src/pages-v2/repo/transform-utils/branch-transform.ts

```diff
@@ -28,3 +33,11 @@ export const transformBranchList = (
     }
   })
 }
+
+export function apiBranches2BranchNames(apiBranches?: ListBranchesOkResponse) {
+  return apiBranches?.map(apiBranch => apiBranch.name ?? '')
+}
+
+export function apiRepository2DefaultBranchName(apiRepo?: RepoRepositoryOutput, defaultBranch = 'main') {
+  return apiRepo?.default_branch ?? defaultBranch
+}
```

Comment:
i believe these should not be needed. we already have a branch store which handles this